### PR TITLE
DBZ-4200 Remove superfluous build triggers

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-4200

### CODE REVIEW

File: .github/workflows/mongodb-oplog-workflow.yml [hunk 1]
```diff
@@ -8,14 +8,11 @@ on:
     paths:
       - 'support/checkstyle/**'
       - 'support/ide-configs/**'
```
Comment (gunnarmorling) [inline .github/workflows/mongodb-oplog-workflow.yml:10]: This one shouldn't be needed either? If you agree, can you rework the commit, force-push, and merge this PR? Thx!

**Review by gunnarmorling — COMMENTED** [review]
Welcome back! One comment inline.

File: .github/workflows/mongodb-oplog-workflow.yml (hunk 1 above)
Comment (ani-sha) [reply .github/workflows/mongodb-oplog-workflow.yml:10]: Agree that the `ide-configs` can be removed. The `checkstyle` will be needed though.

File: .github/workflows/mongodb-oplog-workflow.yml [hunk 2]
```diff
@@ -8,14 +8,11 @@ on:
...
-      - 'support/revapi/**'
       - 'debezium-api/**'
       - 'debezium-assembly-descriptors/**'
       - 'debezium-core/**'
       - 'debezium-embedded/**'
       - 'debezium-connector-mongodb/**'
-      - 'debezium-microbenchmark/**'
-      - 'debezium-e2e-benchmark/**'
       - 'debezium-scripting/**'
```
Comment (ani-sha) [inline .github/workflows/mongodb-oplog-workflow.yml:16]: What about the `debezium-scripting` module?

Comment (gunnarmorling) [reply .github/workflows/mongodb-oplog-workflow.yml:16]: Right, this isn't needed either (it's not a dependency of this module).

**Comment by gunnarmorling** [comment]
Thx!

**Pull request merged**